src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,39 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int COUNT_OF_ARRAY = 10;
+    private K [] keys = (K[]) new Object[COUNT_OF_ARRAY];
+    private V [] values = (V[]) new Object[COUNT_OF_ARRAY];
+    private int storageSize = 0;
```

Comment:
```suggestion
    private int storageSize;
```
int value is 0 by default